proposals/stage-0/individual-membership/README.md

```diff
@@ -0,0 +1,131 @@
+# Individual Membership Program
+>  Stage 0
+
+### Champion
+[Tracy Hinds](https://github.com/hackygolucky)
+
+### Description
+
+The projects of the $FOUNDATION should strive to feel more engaging and welcoming. The Individual Membership program brings together $FOUNDATION projects contributors and users – who are not current contributors – through increased participation, solicited input, and first access to some $FOUNDATION provided benefits. This program codifies a relationship with our ecosystem, and provides an opportunity for the $FOUNDATION, and member companies, to connect with engaged community members through systematic communication building.
+
+#### Program Structure
+
+To engage as many community members as possible, the Individual Membership program will offer three membership tiers that each confer a different level of benefit to the member. Increasing member levels correspond to increased annual fees, increased expectations of the member, and more valuable perks. Members will have the option to adjust their membership level and renew their existing membership through the proposed membership management systems, as described in a following section.
+
+#### Program Rollout
+
+The Individual Membership program will roll out in two phases. 
+
+Phase 1: At launch, the program will only have a single tier. This will provide the $FOUNDATION projects early validation of the program’s utility, give program developers the opportunity to receive feedback on tooling and workflow from members, and ease initial launch overhead for the implementers.
+
+Phase 2: The long term goal of the projects will be to deliver three membership tiers, as described below. The initial membership offering would then be known as Bronze Membership, followed by the addition of the two higher tiers in terms of perks.The delay in Silver and Gold Membership launch allow the program time to recruit corporate member perks and mature enough to provide value.
+
+#### Individual Membership Tiers
+
+##### Initial 2019 relaunch membership/Bronze Member ($12/yr) 
+
+###### Rollout Target: Q4 2019
+
+The affordability of this member class allows for folks who are excited about $FOUNDATION projects to feel part of the $FOUNDATION without putting the barrier to entry too high. This membership level affords participants:
+
+1. The $FOUNDATION member badge – an exclusive digital membership badge and sticker 
+2. Public “thank you” on the $FOUNDATION website 
+3. Invitation to a quarterly town hall meeting with the CPC 
+4. Bi-monthly town hall meeting with individual representation champion 
```

Comment:
I think this wording was initially crafted this way in case a director seat no longer existed or hadn't been triggered by the 2000 member threshhold. The member group can still exist without a director though not ideal, and would likely need some other person(possibly CPC member) to help do the work of gathering feedback and communicating it for the group, since we have written that a base expectation of this group would be connection and input.